cmd/cmd/endpointShow.go

```diff
@@ -96,7 +95,7 @@ func executeGetEndpointCmd(endpointname string) {
 // Print the details of an Endpoint
 // Name, Type, Method, Url, Stats
 // @param Endpoint : Endpoint object
-func printEndpoint(endpoint utils.Endpoint) {
+func printEndpoint(endpoint artifactUtils.Endpoint) {
 
 	fmt.Println("Name - " + endpoint.Name)
```

Comment:
Are the formats from different artifact types different? I don't see data being appended to a table as in APIs. @sajithaliyanage any idea?